README.md

```diff
@@ -1,6 +1,8 @@
 # K-9 Mail
-[![Build Status](https://k9mail.ci.cloudbees.com/job/master/badge/icon)](https://k9mail.ci.cloudbees.com/job/master/)
```

Comment:
Please leave the Cloudbees build status in place. The broken image is a reminder that I need to contact them to get the account fixed :)